src/Graphula.hs

```diff
@@ -168,12 +169,15 @@ instance MonadTrans (GraphulaT n) where
   lift = GraphulaT . lift
 
 instance MonadUnliftIO m => MonadUnliftIO (GraphulaT n m) where
-  {-# INLINE askUnliftIO #-}
-  askUnliftIO = GraphulaT $ withUnliftIO $ \u ->
-    return $ UnliftIO $ unliftIO u . runGraphulaT'
+#if MIN_VERSION_unliftio_core(0, 2, 0)
```

Comment:
I would love if we could avoid CPP, but this `unliftio` transition might make it impossible for the time being.

Reply:
It's too bad that supporting both versions requires CPP 😞

Reply:
In older versions, `askUnliftIO` was an optional member of the type class anyway, so just removing it should work across them.